lang/hr.js

```diff
@@ -141,7 +141,10 @@ var language = {
       "title": "Kineski (pojednostavljeni)"
     },
     "zh-HK": {
-      "title": "Kineski (tradicionalni, Hong Kong)"
+      "title": "Chinese (traditional, Hong Kong)"
```

Comment:
I can't see what has happened here. It looks in Crowdin that the string was only uploaded today so it's yet to be translated, but that isn't the case.

Reply:
Do we need to wait a bit a do a new round trip?

Reply:
I think so, they've done the translation again now, so it just needs proofreading. I'll check again in a couple of days

Reply:
This is now fixed @microbit-carlos